jujugui/static/gui/src/app/components/entity-details/test-entity-details.js

```diff
@@ -85,7 +85,7 @@ describe('EntityDetails', function() {
         changeState={sinon.spy()}
         getEntity={sinon.spy()}
         pluralize={sinon.spy()} />);
-    assert.equal(output.props.children, 'Loading...');
+    assert.equal(output.props.className, 'entity-details');
```

Comment:
`deepEqual` was giving me all sorts of fits in `EntityDetails` - when I visually inspected/drilled down into both objects they looked equal in every regard. The `Spinner` itself had its own tests, so I didn't need to cover that. All that the test in `EntityDetails` really needed to verify was that the `Spinner` was wrapped in something with the `entity-details` class. So I streamlined the test to just check that.